src/actions/task.js

```diff
@@ -46,6 +54,7 @@ export const addNewTask = (newTask, wbsId) => async (dispatch, getState) => {
     _id = res.data._id;
     status = res.status;
     task = res.data;
+    console.log(task);
```

Comment:
Please delete this line. We don't allow any debug lines on our production code base.